atomic_reactor/plugins/pre_stop_autorebuild_if_disabled.py

```diff
@@ -52,13 +52,16 @@ def _is_rebuild_enabled(self):
                               'enabled' if result else 'disabled',
                               self.config_file)
             except configparser.Error:
-                self.log.error('can\'t parse "%s", assuming autorebuild is enabled',
-                               self.config_file)
+                self.log.error('can\'t parse "%s", failing plugin', self.config_file)
+                raise AutoRebuildCanceledException(self.key, 'can\'t parse "%s"' % config_file)
```

Comment:
This method shouldn't be raising exceptions. It should return a bool to indicate whether or not rebuild is enabled.